pkg/evaluator/http.go

```diff
@@ -0,0 +1,128 @@
+package evaluator
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"strings"
+
+	om "github.com/checkr/openmock"
+	"github.com/checkr/openmock/swagger_gen/models"
+	"github.com/fatih/structs"
+	"github.com/labstack/echo/v4"
+)
+
+var performReplyHTTPAction = func(context *om.Context, mock *om.ActionReplyHTTP) (*models.ReplyHTTPActionPerformed, error) {
```

Comment:
feels like some functions from openmock itself can be set to public and reuse here

Reply:
That was my original thought; I think it would need some refactoring of the existing openmock stuff.  For example for HTTP the existing `func (a ActionReplyHTTP) Perform` does stuff like write the message to the echo context.  I think to reuse it we'd need some sort of `ReplyHTTPContext` that would hold the shared things that we fix, and then a `EchoAdapter` and an `EvaluateAdatper` that would do the things specific to actual openmock(reply on the HTTP channel) and evaluator (render into a `ReplyHTTPActionPerformed`).  Let's chat about it; I would like to do this in a well-designed way since it should be the template for adding support for other actions later but it seemed like a fair amount of work to me.